src/core/triggers.systemd.in

```diff
@@ -0,0 +1,28 @@
+#  -*- Mode: makefile; indent-tabs-mode: t -*- */
+#
+#  -*- Mode: rpm-spec; indent-tabs-mode: nil -*- */
+#
+#  This file is part of systemd.
+#
+#  Copyright 2015 Zbigniew Jędrzejewski-Szmek
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+#
+#  systemd is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with systemd; If not, see <http://www.gnu.org/licenses/>.
+
+# The contents of this are an example to be copied into systemd.spec.
+
+%transfiletriggerin -- @systemunitdir@ /etc/systemd/system
```

Comment:
It's a "better safe than sorry" consideration. Some packages install commented-out files in there (systemd itself, sssd, tlp). In principle those files shouldn't matter, but I thought it would be simpler to also react to that than to prove that we don't ever need to. Anyway, since packages *rarely* install anything there, triggering on /etc is not costly.